runtests.py

```diff
@@ -1798,6 +1798,13 @@ def runTest(self):
         source_dirs = ['Cython', 'Demos', 'docs', 'pyximport', 'tests']
 
         import pycodestyle
+
+        @pycodestyle.register_check
+        def breakpoint_check(physical_line):
+            idx = physical_line.find('breakpoint()')
+            if idx != -1:
+                return idx, "Z001 Stray 'breakpoint' call"
```

Comment:
It the needle is really a needle and not a pillar, it's usually quicker to throw an 'in' against the hay stack than to call `str.find()` for all lines that most probably don't contain it.
```suggestion
        def breakpoint_check(physical_line):
            if 'breakpoint()' not in physical_line:
                return None
            idx = physical_line.find('breakpoint()')
            return idx, "Z001 Stray 'breakpoint' call"
```